apps/src/tests/Test3212/BottomTabsScenario.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useCallback, useState } from "react";
+import { SCROLL_EDGE_EFFECT_DEFAULTS, ScrollEdgeEffects, ScrollEdgeEffectsConfigContext } from "./context";
+import { NavigationContainer, NavigationIndependentTree } from "@react-navigation/native";
+import { BottomTabsContainer } from "../../shared/gamma/containers/bottom-tabs/BottomTabsContainer"
+import { Config } from "./Config";
+import { ScrollViewTemplate } from "./ScrollViewTemplate";
+import { ScrollView } from "react-native";
+
+export function BottomTabsScenario() {
+  const [config, setConfig] = useState<ScrollEdgeEffects>({ ...SCROLL_EDGE_EFFECT_DEFAULTS });
+
+  // Add ScrollView for automatic insets which are missing in BottomTabsScreen
+  const ConfigComponent = useCallback(() => <ScrollView><Config title='BottomTabs / scrollEdgeEffects:' /></ScrollView>, []);
```

Comment:
This is fine here, I do not require you to change it, but please notice, that this inline component does not depend on any state nor is directly tied to the `BotomTabsScenario` component ==> it could be just a standalone helper component defined  as standalone function in the bottom of the file making things a little bit cleaner.